src/site/stages/build/process-cms-exports/transformers/node-person_profile.js

```diff
@@ -1,27 +1,39 @@
-const { getDrupalValue } = require('./helpers');
+const { getDrupalValue, isPublished, utcToEpochTime } = require('./helpers');
 
-const transform = entity => ({
+const transform = (entity, { ancestors }) => ({
   entityType: 'node',
   entityBundle: 'person_profile',
   title: `${getDrupalValue(entity.fieldNameFirst)} ${getDrupalValue(
     entity.fieldLastName,
   )}`,
+  entityPublished: isPublished(getDrupalValue(entity.status)),
   fieldBody: getDrupalValue(entity.fieldBody),
   fieldDescription: getDrupalValue(entity.fieldDescription),
   fieldEmailAddress: getDrupalValue(entity.fieldEmailAddress),
   fieldLastName: getDrupalValue(entity.fieldLastName),
   fieldMedia: entity.fieldMedia.length > 0 ? entity.fieldMedia[0] : null,
   fieldNameFirst: getDrupalValue(entity.fieldNameFirst),
-  fieldOffice: entity.fieldOffice[0]
-    ? {
-        entity: {
-          entityLabel: entity.fieldOffice[0].entity.entityLabel,
-          entityType: entity.fieldOffice[0].entity.entityType,
-        },
-      }
-    : null,
+  // Check if fieldOffice is an ancestor of this entity
+  // If so, ignore it, because we'll ignore it in parent transformer
```

Comment:
:thinking: Is this original to this PR [or @theodur's](https://github.com/department-of-veterans-affairs/vets-website/pull/13813#discussion_r469536311)?

Reply:
It's original from Theo's. I merged it since I was having problems with `node-person_profile` and he told me to merge his into mine. It did fix my problem,

Reply:
Maybe I can merge my original PR first since the changes were just approved by @cvalarida, and then you could merge `master`? 😅

Reply:
I'm updating some comments in my original PR, so it won't be exactly the same.

Reply:
yeah, that's fine!